lms/djangoapps/certificates/views/webview.py

```diff
@@ -113,6 +113,14 @@ def _update_certificate_context(context, course, user_certificate, platform_name
         year=date.year
     )
 
+    # Translators:  EEF format of the date
+    date = display_date_for_certificate(course, user_certificate)
```

Comment:
There is no need to call this second time. The `date` variable is already calculated at line 109.

Remove this line, please

Reply:
fixed